src/components/FilterComponents/checkbox/CheckBox.js

```diff
@@ -1,3 +1,17 @@
 /**
  * This CheckBox will be used for showing only caught pokemons on the first page.
- */
\ No newline at end of file
+ */
+
+import './CheckBox.css';
+
+
+function CheckBox () {
```

Comment:
you will need an onchange callback added in props and bind it to the inputs onchange method.